prompting/utils/cleaners.py

```diff
@@ -77,16 +85,13 @@ def apply(self, generation: str) -> str:
         )  # LLMs are good at being formal. Do the same if we remove a prefix.
 
 
-class PrunePostQuestionText(BaseCleaner):
-    def __init__(self, **kwargs):
-        pass
-
+class RemovePostQuestionText(BaseCleaner):
     def apply(
         self,
         generation: str,
         min_pos: Union[int, float] = 5,
         max_pos: Union[int, float] = 0.5,
-        max_questions: int = None,
+        max_questions: int | None = None,
```

Comment:
Same with python 3.9
```suggestion
        max_questions: int | None = None,
```

Reply:
Updated to py3.10